sdk/python/foundation-models/system/finetune/text-generation/summarization_with_text_gen.ipynb

```diff
@@ -132,7 +132,22 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "#### Uncomment one of the below cell if you want to use llama-2-7b, mistral7b, mixtral models."
+    "#### Uncomment one of the below cell if you want to use llama-3-8b, llama-2-7b, mistral7b, mixtral models."
```

Comment:
nit: should we be consistent in our usage of Meta-<model> and casing (llama- vs Llama)?

Reply:
Will change in next PR